src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -190,6 +190,7 @@ namespace OutputReportPredefined {
         s->pdchIntFenUfact = newPreDefColumn(state, s->pdstIntFen, "Glass U-Factor [W/m2-K]");
         s->pdchIntFenSHGC = newPreDefColumn(state, s->pdstIntFen, "Glass SHGC");
         s->pdchIntFenVisTr = newPreDefColumn(state, s->pdstIntFen, "Glass Visible Transmittance");
+        s->pdchIntFenParent = newPreDefColumn(state, s->pdstIntFen, "Parent Surface");
```

Comment:
But this line was missing in the initial table setup, so the parent surface column wasn't there.